scripts/lis_hyperv_platform/main.ps1

```diff
@@ -14,36 +12,83 @@ param(
 $ErrorActionPreference = "Stop"
 
 $scriptPath = Split-Path -Parent $MyInvocation.MyCommand.Definition
-$JobPath = Split-Path -Parent $VHDPath
-
 . "$scriptPath\retrieve_ip.ps1"
+. "$scriptPath\asserts.ps1"
+
+# constants
+$KERNEL_ARTIFACTS_URL = @("{0}/hyperv-daemons_{1}_amd64.deb",
+                         "{0}/linux-headers-{1}_{1}-10.00.Custom_amd64.deb",
+                         "{0}/linux-image-{1}_{1}-10.00.Custom_amd64.deb")
+
+function Prepare-LocalEnv {
+    param(
+        [String] $SharedStoragePath,
+        [String] $JobId
+    )
+
+    $path = "/var/lib/lava/dispatcher/tmp/$JobId"
+    $remotePath = "H:\$JobId"
+    $localPath = "C:$path"
```

Comment:
same with C:\
this requires parameterization in the future. now just put it in a variable.